Resources/Prototypes/_NF/Catalog/Fills/Lockers/suit_storage.yml

```
@@ -0,0 +1,16 @@
+#HARDSUITS
+#Brigmedic's hardsuit
+- type: entity
+  id: SuitStorageBrigmedic
+  parent: SuitStorageBase
+  suffix: Brigmedic
+  components:
+  - type: StorageFill
+    contents:
+        - id: NitrogenTankFilled
+        - id: OxygenTankFilled
+        - id: ClothingOuterHardsuitBrigmedic
+        - id: ClothingMaskBreathMedicalSecurity
+        - id: JetpackSecurityFilled
+  - type: AccessReader
+    access: [["Medical"]]
```

```suggestion
    access: [["Security"]]
```
Security access would probably(?) do better here. Brigmedics spawn with Security and Brig access too, something which normal medical roles don't. Just my two cents, though, I'm not your boss.